### PR TITLE
Removes validation on siteKey and secretKey

### PR DESCRIPTION
**Pull request by Mark53CZ**

### CODE REVIEW

**Comment by uestla** [comment]
Thanks for contributing.

Unfortunatelly this change broke tests (try to run `./vendor/bin/tester tests/ -C` to see). Either the tests or the library should be updated so that the tests pass - can I ask you for it please?

Plus could you maybe remove the `requester` key validation as well?

File: src/ReCaptchaControl/DI/Extension.php [hunk 1]
```diff
@@ -49,8 +49,6 @@ public function loadConfiguration()
 		$config = $this->validateConfig($this->defaults);
 		$builder = $this->getContainerBuilder();
 
-		Validators::assertField($config, 'siteKey', 'string');
-		Validators::assertField($config, 'secretKey', 'string');
 		Validators::assertField($config, 'methodName', 'string');
 		Validators::assertField($config, 'requester', 'string');
```
Comment (uestla) [inline src/ReCaptchaControl/DI/Extension.php]: This could be removed as well.

**Comment by uestla** [comment]
On the other hand, after looking into [Validators documentation](https://doc.nette.org/en/2.4/validators#toc-rules) it is possible to have multiple type assertions like this:

```php
Validators::assertField($config, 'secretKey', 'string|Nette\DI\Statement');
```

I'll try to add that later today and close this ;-)

Thanks.

**Comment by Mark53CZ** [comment]
Perfect, thank you. I'm really busy right now.

**Comment by uestla** [comment]
I falsely accused you of breaking tests, sorry :-)

It weren't actually your changes what broke tests but interfering tests from `tests/manual/` folder due to composer-installed recaptcha.

---

However, the extension should handle `Nette\DI\Statements` now. Could you please try it with `dev-master`?

**Comment by Mark53CZ** [comment]
It works. Nice job :)

**Comment by uestla** [comment]
Thanks! Closing now and tagging HEAD :-)

**Comment by uestla** [comment]
Thanks! Closing now and tagging HEAD :-)

**Pull request closed without merging**